app/src/main/res/values/styles.xml

```diff
@@ -325,6 +325,18 @@
         <item name="android:layout_marginTop">9dp</item>
     </style>
 
+    <style name="DDGButton" parent="@style/Widget.MaterialComponents.Button">
```

Comment:
`DDGButton`, does it mean that this is our new default style for buttons? If so, I would expect the button to have rounded corners like in the concept test.

Reply:
Changing it to be broken site specific.